hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/crypto/validators/CryptoCreateChecks.java

```diff
@@ -179,53 +179,51 @@ private ResponseCodeEnum validateKeyAliasAndEvmAddressCombinations(
             final CryptoCreateTransactionBody op) {
         if (onlyKeyProvided(op)) {
```

Comment:
As long as this method is being touched: it's kind of hard to read - manually verify that _all_ such combinations are treated.  (Better now that there are fewer combinations!)  But this plus all its helper functions - kind of unfortunate.  And see also the if-ladder at `CryptoCreateTransitionLogic.asCustomizer` (~line 211).